larpandoracontent/LArHelpers/LArGeometryHelper.cc

```diff
@@ -483,4 +483,33 @@ float LArGeometryHelper::CalculateGapDeltaZ(const Pandora &pandora, const float
     return gapDeltaZ;
 }
 
+//------------------------------------------------------------------------------------------------------------------------------------------
+
+double LArGeometryHelper::GetSigmaUVW(const Pandora &pandora, const double maxSigmaDiscrepancy) 
+{
```

Comment:
Review, but maybe only here that needs to be single float, rather than double.

Reply:
I've made this change.  The only place that doesn't use a double sigmaUVW are the LArGeometryHelper::MergeTwo/ThreePositions functions.  There functions use a float sigmaUVW, but do the same manipulation to it as the other functions using double sigmaUVW.  

Having done a bit of searching the chi2 in the LArGeometryHelper::MergeTwo/ThreePositions functions is redundant.  There are 2-3 places inside LArContent where these functions are called and nothing is done with the chi2 in any of them.  Therefore, I think we need to review and either:
1) Don't bother calculating chi2.
2) Change these floats chi2 to doubles if we think the chi2 might be needed later.